applications/CoSimulationApplication/python_scripts/solver_wrappers/external_solver_wrapper.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_solver_wrapper import CoSimulationSolverWrapper
+
+# Other imports
+import KratosMultiphysics.CoSimulationApplication.co_simulation_tools as cs_tools
+
+def Create(settings, solver_name):
+    return ExternalSolverWrapper(settings, solver_name)
+
+class ExternalSolverWrapper(CoSimulationSolverWrapper):
+    """This class serves as wrapper for external solvers
```

Comment:
This is an example right ? And not a wrapper for all/many of the possible external solvers ! Is this correct ?

Reply:
done